parlai/crowdsourcing/tasks/model_chat/analysis/compile_results.py

```diff
@@ -0,0 +1,522 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+import json
+import os
+import re
+from datetime import datetime
+from typing import Any, Dict
+
+import numpy as np
+import pandas as pd
+
+from parlai.crowdsourcing.utils.acceptability import AcceptabilityChecker
+
+
+class ModelChatResultsCompiler:
+    """
+    Compile and save results of human+model chats.
+
+    Results will be saved on the level of specific conversations, as well as aggregated
+    up the level of each worker as a whole.
+    """
+
+    PROBLEM_BUCKETS = [
```

Comment:
could we make this a cmd line arg as well? like a comma-separated list of problem buckets?

Reply:
rather than re-writing the analysis file itself

Reply:
Yes, good point - changed